### PR TITLE
[hi] Add content/hi/docs/tasks/tools/install-kubectl-windows.md

### PR DESCRIPTION
**Pull request by vedant-kakde**
Localizes

-  `content/hi/docs/tasks/tools/install-kubectl-windows.md`

### CODE REVIEW

**Comment by vedant-kakde** [comment]
This PR is in continuation of hi localization effort: #29353

/language hi

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
+---
+title: Windows पर kubectl इंस्टॉल और सेट करें
```
Comment (mittalyashu) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:2]:
```suggestion
title: Windows पर kubectl इंस्टॉल और सेटअप करें
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+content_type: task
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: Windows पर kubectl इंस्टॉल करें
+---
+
+## {{% heading "prerequisites" %}}
```
Comment (mittalyashu) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:11]:
```suggestion
## {{% heading "आवश्यकताए" %}}
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+आपको kubectl संस्करण का उपयोग करना चाहिए जो आपके क्लस्टर के एक मामूली संस्करण अंतर के भीतर हो. उदाहरण के लिए,  v{{< skew latestVersion >}} क्लाइंट निम्नलिखित के साथ संवाद कर सकते हैं v{{< skew prevMinorVersion >}}, v{{< skew latestVersion >}}, and v{{< skew nextMinorVersion >}} कण्ट्रोल प्लेन.
+kubectl के नवीनतम संस्करण का उपयोग करने से अप्रत्याशित मुद्दों से बचने में मदद मिलती है.
+
+## Windows पर kubectl इंस्टॉल करें
+
+Windows पर kubectl संस्थापित करने के लिए निम्नलिखित विधियाँ मौजूद हैं:
+
+- [Windows पर curl के माध्यम से kubectl बाइनरी इंस्टॉल करें](#Windows-पर-curl-के-माध्यम-से-kubectl-बाइनरी-इंस्टॉल-करें)
+- [Chocolatey या Scoop का उपयोग करके Windows पर इंस्टॉल करें](#Chocolatey-या-Scoop-का-उपयोग-करके-Windows-पर-इंस्टॉल-करें)
+
```
Comment (mittalyashu) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:22]:
```suggestion
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+   choco install kubernetes-cli
+   ```
+   {{% /tab %}}
+   {{% tab name="scoop" %}}
+   ```powershell
+   scoop install kubectl
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+
```
Comment (mittalyashu) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:90]:
```suggestion
```

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/tasks/tools/install-kubectl-windows.md

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+title: Windows पर kubectl इंस्टॉल और सेटअप करें
+content_type: task
+weight: 10
+card:
+  name: tasks
+  weight: 20
+  title: Windows पर kubectl इंस्टॉल करें
+---
+
+## {{% heading "आवश्यकताए" %}}
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:11]:
```suggestion
## {{% heading "prerequisites" %}}
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+
+1. सत्यापित करें कि प्लगइन सफलतापूर्वक इंस्टॉल हो गया है.
+
+   ```shell
+   kubectl convert --help
+   ```
+
+   यदि आपको कोई त्रुटि नहीं दिखाई देती है, तो इसका मतलब है कि प्लगइन सफलतापूर्वक इंस्टॉल हो गया है.
+
+## {{% heading "आगे क्या होगा" %}}
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:182]:
```suggestion
## {{% heading "whatsnext" %}}
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Can you replace all full stops `.` with purna viram '।` please

**Comment by anubha-v-ardhan** [comment]
This PR is similar to #29461 which is already reviewed and merged. It would be really helpful @vedant-kakde if you implement the same kind of changes here too.

File: content/hi/docs/tasks/tools/install-kubectl-windows.md (hunk 5 above)
Comment (vedant-kakde) [reply content/hi/docs/tasks/tools/install-kubectl-windows.md:11]: Yes, I'll do that.

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+
+1. यह सुनिश्चित करने के लिए परीक्षण करें कि `kubectl` का समान संस्करण डाउनलोड के समान है:
+
+   ```cmd
+   kubectl version --client
+   ```
+
+{{< note >}}
+[Windows के लिए Docker Desktop](https://docs.docker.com/docker-for-windows/#kubernetes) `kubectl` का अपना संस्करण `PATH` में जोड़ता है।
+यदि आपने पहले Docker Desktop स्थापित किया है, तो आपको Docker Desktop इंस्टॉलर द्वारा जोड़े गए एक `PATH` से पहले अपनी `PATH` प्रविष्टि डालने की आवश्यकता हो सकती है या डॉकर डेस्कटॉप के `kubectl` को हटा दें।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:70]:
```suggestion
यदि आपने पहले Docker Desktop स्थापित किया है, तो आपको Docker Desktop इंस्टॉलर द्वारा जोड़े गए एक `PATH` से पहले अपनी `PATH` प्रविष्टि डालने की आवश्यकता हो सकती है या Docker Desktop के `kubectl` को हटा दें।
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+
+   - `True` या `False` परिणाम प्राप्त करने के लिए `-eq` ऑपरेटर का उपयोग करके सत्यापन को ऑटोमेट करने के लिए powershell का उपयोग करें:
+
+     ```powershell
+     $($(CertUtil -hashfile .\kubectl.exe SHA256)[1] -replace " ", "") -eq $(type .\kubectl.exe.sha256)
+     ```
+
+1. अपने `PATH` में बाइनरी जोड़ें।
+
+1. यह सुनिश्चित करने के लिए परीक्षण करें कि `kubectl` का समान संस्करण डाउनलोड के समान है:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:62]:
```suggestion
1. यह सुनिश्चित करने के लिए परीक्षण करें कि `kubectl` संस्करण डाउनलोड के समान है:
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+   {{% /tab %}}
+   {{< /tabs >}}
+
+1. यह सुनिश्चित करने के लिए परीक्षण करें कि आपके द्वारा इंस्टॉल किया गया संस्करण उप-टू-डेट है:
+
+   ```powershell
+   kubectl version --client
+   ```
+
+1. अपनी होम डायरेक्टरी पर नेविगेट करें:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:96]:
```suggestion
1. अपनी होम डायरेक्टरी पर जाएं:
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+
+   ```powershell
+   # If you're using cmd.exe, run: cd %USERPROFILE%
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:99]:
```suggestion
   # यदि आप cmd.exe का प्रयोग कर रहे हैं, तो: cd %USERPROFILE%
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 11]
```diff
@@ -0,0 +1,184 @@
...
+   cd ~
+   ```
+
+1. `.kube` डायरेक्टरी बनाएं:
+
+   ```powershell
+   mkdir .kube
+   ```
+
+1. आपके द्वारा अभी बनाई गई `.kube` डायरेक्टरी में बदलें:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:109]:
```suggestion
1. आपके द्वारा अभी बनाई गई `.kube` डायरेक्टरी में जाएं:
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 12]
```diff
@@ -0,0 +1,184 @@
...
+
+{{< include "included/kubectl-convert-overview.md" >}}
+
+1. कमांड के साथ नवीनतम रिलीज डाउनलोड करें:
+
+   ```powershell
+   curl -LO https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl-convert.exe
+   ```
+
+1. बाइनरी को मान्य करें (ऐच्छिक)
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:149]:
```suggestion
1. बाइनरी को मान्य करें (वैकल्पिक)
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 13]
```diff
@@ -0,0 +1,184 @@
...
+
+नीचे Zsh के लिए ऑटोकम्प्लेशन स्थापित करने की प्रक्रियाएँ हैं, यदि आप इसे Windows पर चला रहे हैं।
+
+{{< include "included/optional-kubectl-configs-zsh.md" >}}
+
+### इंस्टॉल `kubectl convert` प्लगइन
+
+{{< include "included/kubectl-convert-overview.md" >}}
+
+1. कमांड के साथ नवीनतम रिलीज डाउनलोड करें:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:143]:
```suggestion
1. इस कमांड से नवीनतम रिलीज डाउनलोड करें:
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 14]
```diff
@@ -0,0 +1,184 @@
...
+
+   kubectl-convert चेकसम फ़ाइल डाउनलोड करें:
+
+   ```powershell
+   curl -LO https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/kubectl-convert.exe.sha256
+   ```
+
+   चेकसम फ़ाइल से kubectl-convert बाइनरी को मान्य करें:
+
+   - मैन्युअल रूप से कमांड प्रॉम्प्ट का उपयोग करके `CertUtil` के आउटपुट की तुलना डाउनलोडेड चेकसम फ़ाइल से करें:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:159]:
```suggestion
   - मैन्युअल रूप से कमांड प्रॉम्प्ट का उपयोग करके `CertUtil` के आउटपुट की तुलना डाउनलोड किये गये चेकसम फ़ाइल से करें:
```

File: content/hi/docs/tasks/tools/install-kubectl-windows.md [hunk 15]
```diff
@@ -0,0 +1,184 @@
...
+
+     ```cmd
+     CertUtil -hashfile kubectl-convert.exe SHA256
+     type kubectl-convert.exe.sha256
+     ```
+
+   - `True` या `False` परिणाम प्राप्त करने के लिए `-eq` ऑपरेटर का उपयोग करके सत्यापन को ऑटोमेट करने के लिए powershell का उपयोग करें:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/install-kubectl-windows.md:166]:
```suggestion
   - `True` या `False` परिणाम प्राप्त करने और `-eq` ऑपरेटर का उपयोग करके सत्यापन को ऑटोमेट करने के लिए Powershell का उपयोग करें:
```

**Comment by vedant-kakde** [comment]
Thanks for reviewing @anubha-v-ardhan 😄

**Pull request merged**